demos/adjoint_2d_cylindrical/forward.py

```diff
@@ -85,9 +85,9 @@ def step_func(centre, mag, increasing=True, sharpness=50):
     mu_lin *= exp(-ln(Constant(80)) * T)
 
     # Assemble the viscosity expression in terms of velocity u
-    eps = 0.5 * sym(grad(u))
-    epsii = sqrt(inner(eps, eps))
-    sigma_y = 1e4 + 2.0e5 * (rmax - r)
+    eps = sym(grad(u))
+    epsii = sqrt(0.5 * inner(eps, eps) + 1e-10)
+    sigma_y = 1.4142e4 + 2.8284e5 * (rmax - r)
```

Comment:
Should these be e.g. `1e4 * sqrt(2)` and `2e5 * sqrt(2)` or something instead of just floats?